en/08_Changelogs/6.0.0.md

```diff
@@ -58,6 +60,71 @@ All other models used `SearchableDropdownField` for `has_one` relations and `Gri
 - Native indexed PHP arrays can now be passed into templates and iterated over with `<% loop $MyArray %>`. Under the hood they are wrapped in [`ArrayList`](api:SilverStripe\View\ViewableData), so you can get the count using `$Count` and use `<% if $ArrayList %>` as a shortcut for `<% if $ArrayList.Count %>`. Other functionality from `ArrayList` such as filtering and sorting cannot be used on arrays since they don't have keys to filter or sort against.
 - Modules no longer need to have a root level `_config.php` or `_config` directory to be recognised as a Silverstripe CMS module. They will now be recognised as a module if they have a `composer.json` file with a `type` of `silverstripe-vendormodule` or `silverstripe-theme`.
 
+## Dependency changes
+
+### `intervention/image` has been upgraded from v2 to v3 {#intervention-image}
+
+We've upgraded from `intervention/image` v2 to v3. One of the main improvements included in this upgrade is full support for animated GIFs.
+
+If you are directly interacting with APIs from `intervention/image` in your project or module you should check out [their upgrade guide](https://image.intervention.io/v3/introduction/upgrade).
+
+#### Animated vs still images {#intervention-image-animations}
+
+Manipulating animated images takes longer, and results in a larger filesize.
+
+Because of this, the [`ThumbnailGenerator`](api:SilverStripe\AssetAdmin\Model\ThumbnailGenerator) will provide still images as thumbnails for animated gifs by default. You can change that for a given instance of `ThumbnailGenerator` by passing `true` to the [`setAllowsAnimation()`](api:SilverStripe\AssetAdmin\Model\ThumbnailGenerator::setAllowsAnimation()) method. For example, to allow animated thumbnails for `UploadField`:
```

Comment:
This also applies to all other PRs to changelogs - we're essentially duplicating docs on doc.silverstripe.org.  We should just link to those docs instead of duplicating them in the changelog.

Reply:
It's not at all uncommon for us to do this - we often give the most pertinent information directly in the changelog, even if it's also documented elsewhere.

> This also applies to all other PRs to changelogs

I assume you want some amount of this removed, can you please indicate how much so I know how much I want to push back on it? :p

Reply:
> It's not at all uncommon for us to do this - we often give the most pertinent information directly in the changelog, even if it's also documented elsewhere.

Yeah we do ... though just looking at this PR I actually have no idea why we've been doing this. We're quick to call out duplicate documentation in all other scenarios and say "You should just link to the original source" ... I'm not sure why the changelog should be treated any differently.

> I assume you want some amount of this removed, can you please indicate how much so I know how much I want to push back on it? :p

Probably everything not in the first section ``intervention/image` has been upgraded from v2 to v3 {#intervention-image}` - I'd keep that and say that imagick is now the default if it's available and link to the docs

Most of the rest is duplication of "how to configure it", which is basically straight duplication

I'd also remove the "New API" bit, it just seems unnecessary.  Very few people are going to be interested in calling the PHP methods directly, the configuration is much more relevant.  If they really are interested in calling PHP methods directly then there's typehinting available anyway.